pom.xml

```diff
@@ -153,6 +153,16 @@
 						</filter>
 					</filters>
 				</configuration>
+				<executions>
```

Comment:
I think we should just not include such hacks in the pom, especially not at part of this commit, to give exposure to https://github.com/eclipse/tycho/issues/146

Reply:
I didn't know there is already an issue for this, so I wanted to prevent corresponding errors in the m2e poms.
But if you are fine with them, I'm also.
Thanks for the remark.